ui/src/app/job-details/job-details.component.ts

```diff
@@ -40,6 +42,46 @@ export class JobDetailsComponent implements OnInit {
     });
   }
 
+  handleNavUp(): void {
+    if (this.job.extensions.parentJobId) {
+      this.router.navigate(['/jobs/' + this.job.extensions.parentJobId], {
```

Comment:
Should just use a routerLink in the template if possible so that t behaves like a proper link, e.g. ctrl+click does the right thing

Reply:
(in context of #434  - but I guess this did address the bug - opening in a new window)

Reply:
I tried using a `routerLink`, but it wouldn't reload the components in the page, it would only update the URL.